### ISSUE TITLE
Audio crackling when reproducing videos and background music

### ISSUE THREAD

**Issue by sircharlo** [sircharlo/mmm-refactor#68]
Is it just me or it has some audio crackling when reproducing videos and also with background music on?

_Originally posted by @alanfermtz in https://github.com/sircharlo/mmm-refactor/discussions/2#discussioncomment-10000613_

**Comment by sircharlo** [comment]
Just to be clear @alanfermtz, do you mean when playing a video at the same time as background music?

**Comment by alanfermtz** [comment]
No, I notice it when I change from a window to another or clicking my task bar icons, just randomly
I will continue to overview if happen to me again, and try to catch it

**Comment by sircharlo** [comment]
Hmm. Could you give me more info about your system specs?

For example, the output of the portable version of HWiNFO:

![image](https://github.com/sircharlo/mmm-refactor/assets/1275666/812fbcfe-8cfe-4335-b62c-87a2b6a4f307)

**Comment by alanfermtz** [comment]
![image](https://github.com/sircharlo/mmm-refactor/assets/43801184/29932dee-0528-4aa7-94fb-45ba6db81319)

**Comment by ux-git** [comment]
@alanfermtz recording of your screen with system audio on will definitely be helpful in trying to reproduce and troubleshoot the issue.

**Comment by sircharlo** [comment]
I'll add a toggle to disable hardware acceleration in the advanced settings, perhaps this is GPU related.

**Comment by sircharlo** [comment]
After listening to an audio recording, I believe that the issue may be the way that the media is stopped when a video is not yet done. I'll look into updating the mechanism, but I would not consider this a major issue.

**Comment by sircharlo** [comment]
Please re-open if the issue is still present in v24.8.1 @alanfermtz
